### ISSUE TITLE
Icelandic translation

### ISSUE THREAD

**Issue by jokull** [squidfunk/mkdocs-material#2387]
``` jinja
{% macro t(key) %}{{ {
  "language": "is",
  "direction": "ltr",
  "clipboard.copy": "Afrita í klemmuspjald",
  "clipboard.copied": "Afritað í klemmuspjald",
  "edit.link.title": "Ritvinna þessa síðu",
  "footer.previous": "Fyrra",
  "footer.next": "Næsta",
  "footer.title": "Síðufótur",
  "header.title": "Haus",
  "meta.comments": "Athugasemdir",
  "meta.source": "Grunnur",
  "nav.title": "Valmynd",
  "search.config.lang": "is",
  "search.config.pipeline": "",
  "search.config.separator": "[\s\-]+",
  "search.placeholder": "Leit",
  "search.reset": "Hreinsa",
  "search.result.placeholder": "Sláðu inn til að hefja leit",
  "search.result.none": "Engin skjöl fundust",
  "search.result.one": "1 skjal fannst",
  "search.result.other": "# skjöl fundust",
  "search.result.more.one": "1 til viðbótar á þessari síðu",
  "search.result.more.other": "# til viðbótar á þessari síðu",
  "skip.link.title": "Hoppa yfir í efni",
  "source.link.title": "Fara í gagnahirslu (e. repository)",
  "source.revision.date": "Síðasta uppfærsla",
  "tabs.title": "Flipar",
  "toc.title": "Efnisyfirlit"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks for providing!

**Comment by squidfunk** [comment]
Added in f87e32de.

**Comment by squidfunk** [comment]
Released with 7.0.4

**Comment by coliff** [comment]
Heya @squidfunk - there’s a typo in the language name in the changelog

**Comment by squidfunk** [comment]
Woops! Thanks for pointing it out. In German it's called "Island", so this was probably the reason of the mistake. Fixed in 99aa411d.
